.github/pull_request_template.md

```diff
@@ -1,14 +1,15 @@
 ## Summary
-Fixes #{ISSUE}
 
-### Time to review: __x mins__
+Fixes #ISSUE
 
 ## Changes proposed
-> What was added, updated, or removed in this PR.
+
+<!-- (What was added, updated, or removed in this PR.) -->
```

Comment:
we didn't discuss this, but I changed this to comment out the directions so if someone forgets to delete them they don't show up in the final output (but would show up in a commit message if not deleted, so open to the idea of keeping them visible in order to encourage folks to delete them)